src/btf.cpp

```diff
@@ -250,9 +250,16 @@ std::string BTF::c_def(const std::unordered_set<std::string> &set) const
   if (!has_data())
     return std::string("");
 
-  // Definition dumping from modules would require to resolve type conflicts,
-  // so we dump from vmlinux only for now.
+  // Definition dumping from multiple modules would require to resolve type
+  // conflicts, so we allow dumping from a single module or from vmlinux only.
   std::unordered_set<std::string> to_dump(set);
+  if (btf_objects.size() == 2)
```

Comment:
It looks like if `bpf_objects.size() > 2`, then only vmlinux btf is dumped. And user will get an ambiguous error about incomplete/missing definitions. Do we want to have a more explicit error check?

Reply:
Good idea. It'll happen only if probes from multiple modules are used at once, but adding the error message should be relatively easy.